projects/apple/tomviz.bundle.cmake

```diff
@@ -32,7 +27,7 @@ install(CODE "
          USE_SOURCE_PERMISSIONS TYPE DIRECTORY FILES
          \"${install_location}/lib/python2.7/site-packages/\")
 
-    if(${itk_condition_variable})
+    if(${itk_ENABLED})
```

Comment:
That's because this isn't actually CMake code.  It is a string that gets written to a script an executed.  In the context where it is executed itk_ENABLED (or any other variable in the superbuild) has no value, so this is writing the value to the file using variable substitution.  For the if statement to have the ${} in the script that is written I would have to escape the $.